packages/contracts/deployments/README.md

```diff
@@ -90,19 +90,18 @@ Network : __kovan (chain id: 42)__
 |OVM_StateManagerFactory|[0xBcca22E9F5579193E27dD39aD821A03778C44EFA](https://kovan.etherscan.io/address/0xBcca22E9F5579193E27dD39aD821A03778C44EFA)|
 |OVM_StateTransitionerFactory|[0xFD7B9268e790837d393Fd371Ddeb42FE5EC45B54](https://kovan.etherscan.io/address/0xFD7B9268e790837d393Fd371Ddeb42FE5EC45B54)|
 |Proxy__OVM_L1CrossDomainMessenger|[0x4361d0F75A0186C05f971c566dC6bEa5957483fD](https://kovan.etherscan.io/address/0x4361d0F75A0186C05f971c566dC6bEa5957483fD)|
-|Proxy__OVM_L1ETHGateway|[0xB191d67F69e823445cD59e5A88953a82be73b9C6](https://kovan.etherscan.io/address/0xB191d67F69e823445cD59e5A88953a82be73b9C6)|
 |Proxy__OVM_L1StandardBridge|[0x22F24361D548e5FaAfb36d1437839f080363982B](https://kovan.etherscan.io/address/0x22F24361D548e5FaAfb36d1437839f080363982B)|
 |mockOVM_BondManager|[0xD6143943447DFf503d948Fba3D8af3d4Df28f45c](https://kovan.etherscan.io/address/0xD6143943447DFf503d948Fba3D8af3d4Df28f45c)|
 <!--
 Implementation addresses. DO NOT use these addresses directly.
 Use their proxied counterparts seen above.
 
-OVM_L1CrossDomainMessenger: 
+OVM_L1CrossDomainMessenger:
  - 0xa172330C2E6Ec2bF04662Bb9b67ae857910b7f7f
  - https://kovan.etherscan.io/address/0xa172330C2E6Ec2bF04662Bb9b67ae857910b7f7f)
-OVM_L1ETHGateway: 
- - 0x25bb69ee5665536Ce6aeb51094F0bed9e4DACc30
- - https://kovan.etherscan.io/address/0x25bb69ee5665536Ce6aeb51094F0bed9e4DACc30)
+OVM_L1StandardBridge:
+ - 0x8293ab0dc701a1387031a13786276f1baa3fcd4e
+ - https://kovan.etherscan.io/address/0x8293ab0dc701a1387031a13786276f1baa3fcd4e)
 -->
```

Comment:
I've noticed that the `OVM_L1StandardBridge` contract isn't verified - how was it deployed? I don't see an artifact for it

Reply:
Good question. We should have an artifact for it cc @elenadimitrova and @maurelian.

Reply:
Honestly, I don't know, I was not involved in the deployment process.

Reply:
I need to write up an issue for this but it's unrelated to this PR.